# example of online deployment parameters set up

trained: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/Snapshots/stack-fc/yaw/LSTM_deep-regression-3-64-16-27_iter_550000.caffemodel

online_model: /home/leonida/ThesisCode/LSTM_on_Create2/src/neural_network_planner/NetModels/LSTM/deep_stack-fc_online.prototxt

GPU: false

control_rate: 10 # Hertz

pos_update_threshold: 0.01 # meters

cruise_linear_vel: 0.3

target_tolerance: 0.5

averaged_ranges_size: 24

# if it is a classification net
multiclass: false

# classes yaw resolution
yaw_resolution: 30

# add the previous action to the input state
steer_feedback: true

# translational velocity limits
cruise_vel: 0.3
min_cruise_vel: 0.1

# rotational velocity limits
min_rotate_vel: -1 
max_rotate_vel: 1
rotate_vel_res: 0.05

# if true publish state ranges as lines, false means only points 
show_lines: true

time_sequence: 10

logs_path: /home/leonida/ThesisCode/LSTM_on_Create2/logs/ 
